CodeU_Assignement_5/src/codeu_assignement_5/KemUnknownLanguage.java

```diff
@@ -0,0 +1,145 @@
+package codeu_assignement_5;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author Karine
+ */
+public class KemUnknownLanguage {
+    /*
+     * Input: a dictionary (a list of words in lexicographic order) of all words in an unknown/invented language, 
+    *  Output: alphabet (an ordered list of characters) of that language, letters we cannot decide are appended in the end.
+    */
```

Comment:
While you can express this purely in terms of "input" and "output". I think it's better to say what it's doing and then express the input/output in terms of that. e.g.

> Infer an alphabet (ordered list of characters) from a dictionary (list of words in lexicographic order).
> Input: ...
> Output: ...

Note that "letters we cannot decide are appended in the end" is confusing - it sounds like all ambiguous characters get put at the end, but ambiguity can occur anywhere in the alphabet. D, XA, XC, XD, YA, YB, YD would mean that we know that B and C are between A and D (i.e. can't be appended at the end), but we don't know the relative ordering of B and C.